.github/workflows/test-assistant-pr-review.yml

```diff
@@ -0,0 +1,44 @@
+name: Send PRs that require review to Slack
+
+on:
+  pull_request_target:
+    types: [review_requested]
+  issue_comment:
+    types: [created]
+
+permissions: {}
+
+jobs:
+  send-pr-to-slack-for-review:
+    runs-on: ubuntu-20.04
+    env: 
+      WOO_PR_TESTING_REVIEW_TEAM: ${{ secrets.WOO_PR_TESTING_REVIEW_TEAM }}
+    steps:
+      - name: Send ping to Slack for a new review
+        uses: archive/github-actions-slack@d9dae40827adf93bddf939db6552d1e392259d7d
+        id: notify-review
+        if: |
+          github.event_name == 'pull_request' &&
+          github.event.action == 'review_requested' &&
+          github.event.requested_team.slug == env.WOO_PR_TESTING_REVIEW_TEAM
+        with:
+          slack-bot-user-oauth-access-token: ${{ secrets.TEST_ASSISTANCE_BOT_TOKEN }}
+          slack-channel: ${{ secrets.WOO_PR_TESTING_REVIEW_SLACK_CHANNEL }}
+          slack-text: |
+              <!subteam^${{ secrets.WOO_PR_TESTING_REVIEW_TEAM_ID }}> was asked to review this PR:
+              <${{ github.event.pull_request.html_url }}|${{ github.event.pull_request.title }}> :thread:
+          slack-optional-unfurl_links: false
+          slack-optional-unfurl_media: false
+          
+      - name: Send ping to Slack about a new comment
+        uses: archive/github-actions-slack@d9dae40827adf93bddf939db6552d1e392259d7d
+        id: notify-comment
+        if: github.event_name == 'issue_comment' && contains(github.event.comment.body, env.WOO_PR_TESTING_REVIEW_TEAM)
```

Comment:
I created this issue comment but the workflow wasn't triggered. 
https://github.com/woocommerce/woocommerce/pull/41067#issuecomment-1787092943

Any idea why? The if condition looks correct though.

Reply:
Yep, this is explained in the [other thread](https://github.com/woocommerce/woocommerce/pull/41067#discussion_r1377543067), the comment you added here is different to the one added in the PR page (without doing any review), and both are triggered by different events.

Reply:
Right, it makes sense now. Thanks :). Will mark this as resolved.